Mathlib/LinearAlgebra/TensorProduct/Associator.lean

```diff
@@ -343,26 +343,26 @@ lemma rTensor_lTensor_comp_assoc_symm (x : M →ₗ[R] N) :
 end LinearMap
 
 namespace Equiv
-variable {R A B : Type*} [CommSemiring R]
+variable {R A A' B B' C C' : Type*} [CommSemiring R]
+  [AddCommMonoid A'] [AddCommMonoid B'] [AddCommMonoid C'] [Module R A'] [Module R B'] [Module R C']
 
 variable (R) in
 open TensorProduct in
--- TODO: Is there a better place for this?
```

Comment:
I'd be inclined to keep this comment; these lemmas do feel a bit weird to me, but that's out of scope for this PR.